components/conversations/Comment.js

```diff
@@ -314,6 +360,93 @@ const Comment = ({ comment, canEdit, canDelete, withoutActions, maxCommentHeight
             )
           }
         </InlineEditField>
+        {Object.entries(selectedReactions).map(([emoji, count]) => {
+          return (
+            <StyledLink
```

Comment:
If it does not have an `href`, maybe we can use something else than a link for that? What about `Span`?